tests/testAstrometry.py

```diff
@@ -67,12 +67,11 @@ def makeObservationMetaData():
 
     obsDict['Opsim_expmjd'] = mjd
     radius = 0.1
-    phoSimMetadata = OrderedDict([
+    phoSimMetaData = OrderedDict([
                       (k, (obsDict[k],numpy.dtype(type(obsDict[k])))) for k in obsDict])
 
-    obs_metadata = ObservationMetaData(boundType = 'circle', unrefractedRA = numpy.degrees(centerRA),
-                                       unrefractedDec = numpy.degrees(centerDec), boundLength = 2.0*radius,
-                                       phoSimMetadata=phoSimMetadata, site=testSite)
+    obs_metadata = ObservationMetaData(boundType = 'circle', boundLength = 2.0*radius,
```

Comment:
There's another large branch I'm working on for sims_coordUtils.  I will make this change a part of that (so that I don't enter rebase hell when I go back to working on the other branch)